src/client/parse_func.c

```diff
@@ -1220,7 +1220,7 @@ xddfunc_endtoend(xdd_plan_t *planp, int32_t argc, char *argv[], uint32_t flags)
 						} else {
 			    			tdp->td_e2ep->e2e_address_table[tdp->td_e2ep->e2e_address_table_next_entry].port_count = 0;
 						}
-#if HAVE_CPU_SET_T && HAVE_NUMA_NODE_TO_CPUS && HAVE_NUMA_ALLOCATE_CPUMASK
+#if HAVE_CPU_SET_T && HAVE_ENABLE_NUMA
```

Comment:
This is probably another dumb question, but where are we checking and setting `HAVE_CPU_SET_T`? I am just wondering if I am missing something. Also, I need this to be merged into the `xdd-2.0` branch before I can finish up  this PR https://github.com/MigeljanImeri/xdd/pull/17.

Reply:
Ah okay. I don't know how I missed that lol. Alright, I am going to approve this so we can get it merged and I can finish the other PR up.